.prettierrc.json

```diff
@@ -3,5 +3,5 @@
   "useTabs": false,
   "semi": false,
   "singleQuote": true,
-  "trailingComma": "es5"
+  "trailingComma": "ef3"
```

Comment:
[`ef3`/`es3` is not a valid mode](https://prettier.io/docs/en/options.html#trailing-commas). Also, why should we change the way our trailing commas behave?